internal/services/team/create.go

```diff
@@ -2,10 +2,24 @@ package team
 
 import (
 	"context"
+	"fmt"
+	"log"
+	"time"
+
 	"github.com/jairogloz/go-l/internal/domain"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
 func (s Service) Create(ctx context.Context, team *domain.Team) (err error) {
-	//TODO implement me
-	panic("implement me")
+	now := time.Now().UTC()
+	team.CreatedAt = primitive.DateTime(now.UnixNano() / int64(time.Millisecond))
```

Comment:
Una disculpa, el tipo de createdAt y updatedAt debía ser *time.Time, con esta sugerencia debería funcionar.

```suggestion
	team.CreatedAt = &now
	team.UpdatedAt = &now
```